### DISCUSSION TITLE
Group 3 Project Ideas

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#2] [General]
I'm just waiting for Professor Efrain to get back to us on the questions.  

Hope you are doing well.  Just starting a discussion.

**Comment by tay-baum** [comment]
I saw the Professor suggested landing on one movie or show. Any thoughts? I know we discussed Parks & Rec over zoom. If we want to do a movie, maybe something that's come out more recently? I've seen Thor and Top Gun. I'm also a big Lord of the Rings fan and we could always do something less recent. Just wanted to get some ideas going!

I also saw he suggested doing wireframes. I wouldn't mind working on some basic ideas for those (after we land on a movie or show and what pages we'd like to do) since Matthew started the project and Shannon is working on the style guide. They can at least be a starting point that we can discuss more!

**Reply by matty10s** [reply]
Hey Team, I apologize for my slow response this week, I lost my job it’s weighing me down. Trying to get back on the horse.

That would be great Taylor! we can do some wireframes for the project this weekend would be a good start!
